Backpack/BarChart/Classes/BarChartBar.swift

```diff
@@ -0,0 +1,181 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+@objcMembers
+@objc(BPKBarChartBar)
+public final class BPKBarChartBar: UIControl {
+
+    let backgroundView: UIView = UIView()
+    let barView: UIView = UIView()
+    let miniFlareView: BPKMiniFlareView = BPKMiniFlareView()
+    let titleLabel: BPKLabel = BPKLabel(fontStyle: .textBaseEmphasized)
```

Comment:
Can I suggest we use lazy getters for these in which we do all the setup required i.e.

```swift
    lazy var titleLabel: BPKLabel = {
        let label = BPKLabel(fontStyle: .textBaseEmphasized)
        label.translatesAutoresizingMaskIntoConstraints = false

        return label
    }()
```